homeassistant/components/xiaomi_miio/__init__.py

```diff
@@ -75,6 +85,56 @@ def get_platforms(config_entry):
     return []
 
 
+async def async_create_miio_device_and_coordinator(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
+):
+    """Set up a data coordinator and one miio device to service multiple entities."""
+    model = entry.data[CONF_MODEL]
+    host = entry.data[CONF_HOST]
+    token = entry.data[CONF_TOKEN]
+    name = entry.title
+    device = None
+    if model not in MODELS_HUMIDIFIER:
+        return
```

Comment:
Something like:
```python
_LOGGER.error(
        "Unsupported device found! Please create an issue at "
        "https://github.com/syssi/xiaomi_airpurifier/issues "
        "and provide the following data: %s",
        model,
)
```

Is in e.g. fan.py. We should not put this in __init__.py for now because then we should check all device models, we log an error in de platform code instead. The URL for issue e.g. might be some confusing.
Since async_create_miio_device_and_coordinator is always called as a generic device setup it should log and an error here, since that would also impact other device classes. There where we want to use a coordinator we can add additional models here. If we want to centralize this we could use `get_platforms()`, and log an error there there, but we should do remove the logging in the platform code.

Reply:
I have added a line that logs an error if no platform was found to be set up. I think that should cover it.